packages/ethereum-erc20-provider/lib/EthereumERC20Provider.js

```diff
@@ -1,20 +1,25 @@
-import Provider from '../../Provider'
-import { padHexStart } from '../../crypto'
-import { ensureAddressStandardFormat, ensureHexEthFormat } from './EthereumUtil'
 import { BigNumber } from 'bignumber.js'
 
-export default class EthereumERC20Provider extends Provider {
+import Provider from '@liquality/provider'
+import { padHexStart } from '@liquality/crypto'
+import {
+  ensureAddressStandardFormat,
+  ensureHexEthFormat
+} from '@liquality/ethereum-utils'
+
+const SOL_TRANSFER_FUNCTION = '0xa9059cbb' // transfer(address,uint256)
+const SOL_BALACE_OF_FUNCTION = '0x70a08231' // balanceOf(address)
+
+export default class EthereumErc20Provider extends Provider {
   constructor (contractAddress) {
     super()
     this._contractAddress = ensureAddressStandardFormat(contractAddress)
   }
 
   generateErc20Transfer (to, value) {
-    // function transfer(address to, uint tokens) public returns (bool success)
-    const functionSignature = 'a9059cbb'
     const encodedAddress = padHexStart(ensureAddressStandardFormat(to), 64)
     const encodedValue = padHexStart(value.toString(16), 64)
-    return functionSignature + encodedAddress + encodedValue
+    return SOL_TRANSFER_FUNCTION.substring(2) + encodedAddress + encodedValue
```

Comment:
Strange to see these changes here. Just checking if this is some git problem?

Reply:
No. I have made these changes. Making functional signature more readable.